gwcs/utils.py

```diff
@@ -85,9 +85,10 @@ def get_values(units=None, *args):
         Quantity inputs.
     """
     if units is not None:
-        return [a.to_value(unit) for a, unit in zip(args, units)]
+        result = [a.to_value(unit) for a, unit in zip(args, units)]
     else:
-        return [a.value for a in args]
+        resutl = [a.value for a in args]
```

Comment:
```suggestion
        result = [a.value for a in args]
```

Reply:
Good catch! Should see why tests did not catch it. Thanks!